src/Debug.h

```diff
@@ -65,6 +65,61 @@ class debug {
     static int debug_level();
 };
 
+/** Allow easily printing the contents of containers, or std::vector-like containers,
+ *  in debug output. Used like so:
+ *        std::vector<Type> arg_types;
+ *        debug(4) << "arg_types: " << ConPrint(arg_types) << "\n";
+ * Which results in output like "arg_types: { uint8x8, uint8x8 }" on one line.
+ * "ConPrint" Stands for "Container Print." */
+template<typename T>
+struct ConPrint {
```

Comment:
How about `SpanPrint` since something `std::span` like is arguably what we want?

Reply:
SpanPrint is good, or SeqPrint. Con suggested "console" to me, which confused me until I read the code.

Reply:
Done